frontend/lib/pages/issue-pages.tsx

```diff
@@ -121,17 +121,7 @@ type IssueAreaLinkProps = {
 
 function IssueAreaLink(props: IssueAreaLinkProps): JSX.Element {
   const { area, label } = props;
-
-  const [hadViewedModal, updateModalStatus] = useState(false);
-  // useEffect here needs to run at any state update as we need to consistently check 
-  // whether the browserStorage has changed
-  //
-  // TODO: eslint doesn't like the following line because it can cause an infinite
-  // chain of updates, which is a valid concern. I'm disabling the line for now
-  // because I want to enable eslint going forward, but we should definitely revisit
-  // it! -AV
-  // eslint-disable-next-line
-  useEffect( () => updateModalStatus(browserStorage.get('hasViewedCovidRiskModal') || false ));
+  const {hasViewedCovidRiskModal} = useBrowserStorage()[0];
```

Comment:
This is the one part of the structure of this new change that I don't quite understand: why is it important for `useBrowserStorage()` to return an array, and thus why we need to access its first element?

Reply:
Oh, so I wanted `useBrowserStorage()` to work like `useState()`, where it actually returns a `[state, setState]` tuple, only in this case it's `[browserStorage, updateBrowserStorage]` instead.  I think it might read better if it were like this:

```js
const [{hasViewedCovidRiskModal}, updateBrowserStorage] = useBrowserStorage();
```

in that case it'd be clear what `updateBrowserStorage` is for, the only funky part is that it would never actually be used, since this code doesn't actually need it...  I don't _think_ that should cause a linting error but I'm not sure... anyways if it doesn't cause a linting error and it makes sense to you, I can go ahead and change it!

Reply:
Urf, unfortunately the linter doesn't like us naming a variable but not using it, which is annoying :(  I guess I will leave a comment to make it clearer though.